src/git_changelog/cli.py

```diff
@@ -226,6 +226,14 @@ def get_parser() -> argparse.ArgumentParser:
         dest="parse_trailers",
         help="Parse Git trailers in the commit message. See https://git-scm.com/docs/git-interpret-trailers. Default: %(default)s.",
     )
+    parser.add_argument(
+        "--omit-empty",
+        "--omit-empty-versions",
```

Comment:
Let's keep only one long option, and add a short one:

```suggestion
        "--E",
        "--omit-empty-versions",
```

Reply:
Done